src/test/java/api/loans/CheckInByBarcodeTests.java

```diff
@@ -427,18 +427,7 @@ public void patronNoticeOnCheckInIsSentWhenCheckInLoanNoticeIsDefinedAndLoanExis
 
     Awaitility.await()
       .atMost(1, TimeUnit.SECONDS)
-      .until(patronNoticesClient::getAll, hasSize(1));
-    List<JsonObject> sentNotices = patronNoticesClient.getAll();
-
-    Map<String, Matcher<String>> noticeContextMatchers = new HashMap<>();
-    noticeContextMatchers.putAll(TemplateContextMatchers.getUserContextMatchers(james));
-    noticeContextMatchers.putAll(TemplateContextMatchers.getItemContextMatchers(nod, true));
-    noticeContextMatchers.putAll(TemplateContextMatchers.getLoanContextMatchers(checkInResponse.getLoan()));
-    noticeContextMatchers.put("loan.checkedInDate",
-      withinSecondsAfter(Seconds.seconds(10), checkInDate));
-    MatcherAssert.assertThat(sentNotices,
-      hasItems(
-        hasEmailNoticeProperties(james.getId(), checkInTemplateId, noticeContextMatchers)));
+      .until(patronNoticesClient::getAll, hasSize(0));
```

Comment:
Does `until` eagerly finish if the criteria is met? If so, this will pass immediately and not really demonstrate the intended behaviour.

Reply:
updated

Reply:
What changes have been made to address this, as far as I can tell the `until` is still being used, which will succeed eagerly?

Reply:
until has been removed